<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TMClass</class>
 <widget class="QMainWindow" name="TMClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>727</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTabWidget" name="tabMain">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Tasks</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QTreeView" name="treeView">
           <property name="dragEnabled">
            <bool>true</bool>
           </property>
           <property name="dragDropMode">
            <enum>QAbstractItemView::DragDrop</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="animated">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cbHideDone">
           <property name="text">
            <string>Hide done</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Fast filter</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="leFastFilter"/>
           </item>
           <item>
            <widget class="QPushButton" name="btnShownCategories">
             <property name="text">
              <string>Shown categories</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Day activities</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Current activity</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="lblCurrentActivity">
             <property name="text">
              <string>No activity</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="btnToTasks">
             <property name="toolTip">
              <string>Jump to task in task list</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="label_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Started</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="lblActivityStarted">
             <property name="text">
              <string>----.--.-- --:--</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Interrupts</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="lblInterrupts">
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="btnAddInterrupt">
               <property name="toolTip">
                <string>Add interrupt</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Activities</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListView" name="lvActivities"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QToolBox" name="toolBox">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>376</width>
          <height>337</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Notes</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QTextEdit" name="Notes"/>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Create time</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="lblCreateTime">
             <property name="text">
              <string>&lt;no available&gt;</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QCheckBox" name="cbStartedTime">
             <property name="text">
              <string>Start time</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QDateTimeEdit" name="teStartTime">
             <property name="displayFormat">
              <string>yyyy.MM.dd H:mm</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Finish time</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="lblFinishTime">
             <property name="text">
              <string>&lt;not yet finished&gt;</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Planned time</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="lePlannedTime"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Real time brutto</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLabel" name="lblRealTimeBrutto">
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>&lt;no available&gt;</string>
             </property>
             <property name="scaledContents">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Real time netto</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLabel" name="lblRealTimeNetto">
             <property name="text">
              <string>&lt;no available&gt;</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Categories</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLineEdit" name="le_Categories"/>
             </item>
             <item>
              <widget class="QToolButton" name="btnEditCategories"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>376</width>
          <height>337</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Add activity</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Start time</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QDateTimeEdit" name="teActivityStartTime">
             <property name="displayFormat">
              <string>yyyy.MM.dd H:mm</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QToolButton" name="btnUpdateTime">
             <property name="toolTip">
              <string>Change time to current</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Name</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" colspan="2">
            <widget class="QLineEdit" name="leActivityName">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Type</string>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="leftMargin">
             <number>5</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>5</number>
            </property>
            <item>
             <widget class="QRadioButton" name="rbActivityTask">
              <property name="text">
               <string>Task activity</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rbActivityOther">
              <property name="text">
               <string>Separate activity</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnNewActivity">
           <property name="text">
            <string>New activity</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Last activities</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListView" name="lvLastActivities"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>727</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionRestore"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionSync"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTasks">
    <property name="title">
     <string>Tasks</string>
    </property>
    <addaction name="actionAdd"/>
    <addaction name="actionAddChild"/>
    <addaction name="actionDel"/>
    <addaction name="separator"/>
    <addaction name="actionEditCategories"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTasks"/>
  </widget>
  <action name="actionAdd">
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="actionAddChild">
   <property name="text">
    <string>AddChild</string>
   </property>
  </action>
  <action name="actionDel">
   <property name="text">
    <string>Del</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionRestore">
   <property name="text">
    <string>Restore</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSync">
   <property name="text">
    <string>Sync</string>
   </property>
  </action>
  <action name="actionEditCategories">
   <property name="text">
    <string>EditCategories</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>treeView</tabstop>
  <tabstop>tabMain</tabstop>
  <tabstop>rbActivityTask</tabstop>
  <tabstop>rbActivityOther</tabstop>
  <tabstop>leActivityName</tabstop>
  <tabstop>btnNewActivity</tabstop>
  <tabstop>teActivityStartTime</tabstop>
  <tabstop>Notes</tabstop>
  <tabstop>btnToTasks</tabstop>
  <tabstop>lvActivities</tabstop>
 </tabstops>
 <resources>
  <include location="tm.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>activated()</signal>
   <receiver>TMClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbStartedTime</sender>
   <signal>toggled(bool)</signal>
   <receiver>TMClass</receiver>
   <slot>slot_SetStartTime()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>417</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>363</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnAddInterrupt</sender>
   <signal>clicked()</signal>
   <receiver>TMClass</receiver>
   <slot>slot_AddInterrupt()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>363</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSync</sender>
   <signal>activated()</signal>
   <receiver>TMClass</receiver>
   <slot>slot_Sync()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>363</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionEditCategories</sender>
   <signal>activated()</signal>
   <receiver>TMClass</receiver>
   <slot>slot_TotalCategories()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>363</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnEditCategories</sender>
   <signal>clicked()</signal>
   <receiver>TMClass</receiver>
   <slot>slot_Categories()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>693</x>
     <y>387</y>
    </hint>
    <hint type="destinationlabel">
     <x>363</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnShownCategories</sender>
   <signal>clicked()</signal>
   <receiver>TMClass</receiver>
   <slot>slot_SelectShowCats()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>419</y>
    </hint>
    <hint type="destinationlabel">
     <x>363</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leFastFilter</sender>
   <signal>textChanged(QString)</signal>
   <receiver>TMClass</receiver>
   <slot>slot_FastFilter(const QString&amp;)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>134</x>
     <y>419</y>
    </hint>
    <hint type="destinationlabel">
     <x>363</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
